templates/api-server.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "api-server",  
+  "description": "A LoopBack API server with local User auth",
+  "package": {
+    "version": "1.0.0",
+    "main": "server/server.js",
+    "scripts": {
+      "lint": "eslint .",
+      "start": "node .",
```

Comment:
>That would make sense to me only if we included some sample (smoke?) tests in the template too. If you think this is worth pursuing, then please open a new GH issue to discuss.

@bajtos I was thinking along the same lines (add smoke tests in the templates too). Whether the time is now is up for debate. 

@deepakrkris Can you add a summary of the Slack discussion to the bottom of this PR and your reasoning as for landing this now and what you intend to do in the future so people reading this can follow up? Maybe open up a new issue with a discussion label to gather some community feedback on the json vs js notes and @bajtos's input from above together?